base_layer/core/src/validation/helpers.rs

```diff
@@ -20,15 +20,18 @@
 // WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
 // USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+#[cfg(feature = "base_node")]
 use log::*;
-use tari_crypto::tari_utilities::{epoch_time::EpochTime, hex::Hex};
+#[cfg(feature = "base_node")]
+use tari_crypto::tari_utilities::epoch_time::EpochTime;
```

Comment:
Rather reference tari_utilities directly. Why would this need to be feature flagged?